ios/gamma/split-view/RNSSplitViewScreenShadowStateProxy.mm

```diff
@@ -44,7 +44,13 @@ - (void)updateShadowStateWithFrame:(CGRect)frame
 
   if (!CGRectEqualToRect(frame, _lastScheduledFrame)) {
     auto newState = react::RNSSplitViewScreenState{RCTSizeFromCGSize(frame.size), RCTPointFromCGPoint(frame.origin)};
-    _state->updateState(std::move(newState));
+    _state->updateState(
+        std::move(newState),
+#if REACT_NATIVE_MINOR_VERSION >= 82
+        facebook::react::EventQueue::UpdateMode::unstable_Immediate
+#endif
+    );
```

Comment:
Will this compile on 0.81? There is a trailing comma in line 48. Didn't know C++ allows for these. TIL I guess

Reply:
I checked it and you're right here, 
```cpp
void foo(int a, int b=2) {
    std::cout << a << " " << b << std::endl;
}

int main() {
    foo(1,); // fails
    return 0;
}
```
reverting last commit